selftests/functional/test_interrupt.py

```diff
@@ -159,7 +159,10 @@ def wait_until_no_goodtest():
                         if old_psutil:
                             cmdline_list = psutil.Process(p).cmdline
                         else:
-                            cmdline_list = psutil.Process(p).cmdline()
+                            try:
+                                cmdline_list = psutil.Process(p).cmdline()
+                            except psutil.AccessDenied:
+                                cmdline_list = []
```

Comment:
This skips only the process that caused the `AccessDenied` exception, so yep, looks good.